test/verify/check-sosreport

```diff
@@ -35,12 +35,8 @@ class TestSOS(MachineCase):
         b = self.browser
         m = self.machine
 
-        # HACK: sos ships wrong sos.conf, we need to mkdir it, see https://bugzilla.redhat.com/show_bug.cgi?id=1903281
-        m.execute("mkdir -p /etc/sos")
```

Comment:
This [still happens on c8s](https://artifacts.dev.testing-farm.io/43902888-1adf-4c1c-82e5-c855afa5b497/), so the mkdir needs to be put back. The `write_file()` can stay, of course.

Reply:
Do we need to report this somewhere? It worked on rhel-8-6 so I'm kinda suprised.

Reply:
Not sure if it's to be considered a bug, but if you think so, bugzilla away :grin: